packages/webview_flutter/webview_flutter_android/lib/src/android_webview_controller.dart

```diff
@@ -1047,4 +1066,12 @@ class AndroidNavigationDelegate extends PlatformNavigationDelegate {
   Future<void> setOnUrlChange(UrlChangeCallback onUrlChange) async {
     _onUrlChange = onUrlChange;
   }
+
+  /// Invoked when a render process has exited.
+  Future<void> setOnRenderProcessGone(
```

Comment:
This method should probably be moved into `WebViewController`. There could be a scenario where someone uses a single `NavigationDelegate` for multiple `WebViewController`s and then they would not know which one had their render process exited.

Reply:
I agree, but I don't see a simple solution to this problem. `onRenderProcessGone` is defined in the `WebViewClient` https://developer.android.com/reference/android/webkit/WebViewClient#onRenderProcessGone(android.webkit.WebView,%20android.webkit.RenderProcessGoneDetail), which is initialized in the `AndroidNavigationDelegate`. It seems that the `WebViewClient` can be used for multiple `WebView` when calling `setPlatformNavigationDelegate`.

Reply:
Actually, now that I think about it, this is true for every callback method in `WebViewClient`. `NavigationDelegate` doesn't indicate which `WebViewController` is receiving the callback anyways, so a user wouldn't reuse it with that intention. You can just ignore my comment on this.